### PR TITLE
[WR-302] include jira_url & add TestPAMJiraPlugin

### PR DESCRIPTION
**Pull request by pirtleshell**

### CODE REVIEW

**Review by efabens — APPROVED** [review]

File: identityClient/identityClient.go [hunk 1]
```diff
@@ -1346,3 +1346,14 @@ func (c *E3dbIdentityClient) UpdatePAMJiraPlugin(ctx context.Context, params Upd
 	err = e3dbClients.MakeSignedServiceCall(ctx, c.requester, request, c.SigningKeys, c.ClientID, &plugin)
 	return plugin, err
 }
+
+// TestPAMJiraPlugin tests a Jira integration's credentials
```
Comment (galxy25) [inline identityClient/identityClient.go:1350]: how is the test performed / what does success mean?

File: identityClient/api.go [hunk 2]
```diff
@@ -1196,13 +1196,15 @@ type PAMJiraPlugin struct {
 	ID                   int64  `json:"id"`
 	AutomationAuthHeader string `json:"automation_auth_header"`
 	BotUserEmail         string `json:"bot_user_email"`
+	JiraURL              string `json:"jira_url"`
```
Comment (galxy25) [inline identityClient/api.go:1199]: JiraURl is very abstract / hiding some implicit context you have now....suggest something more explicit like `JiraHostURL` or `JiraAPIURL`

File: identityClient/api.go [hunk 3]
```diff
@@ -1219,11 +1221,15 @@ type DeletePAMJiraPluginRequest = PAMJiraPluginByIDRequest
 // DescribePAMJiraPluginRequest wraps necessary data for describing an existing Jira integration
 type DescribePAMJiraPluginRequest = PAMJiraPluginByIDRequest
 
+// TestPAMJiraPluginRequest wraps necessary data for testing an existing Jira integration's credentials
+type TestPAMJiraPluginRequest = PAMJiraPluginByIDRequest
```
Comment (galxy25) [inline identityClient/api.go:1225]: test is a confusing prefix for this struct and method, prefer something like `Ping` or `HealthCheck`

**Review by galxy25 — CHANGES_REQUESTED** [review]
see comments

File: identityClient/identityClient.go (hunk 1 above)
Comment (pirtleshell) [reply identityClient/identityClient.go]: updated comment 👍

File: identityClient/api.go (hunk 3 above)
Comment (pirtleshell) [reply identityClient/api.go]: updated everything to be "ping"

**Review by galxy25 — APPROVED** [review]

**Pull request merged**